### PR TITLE
Add intro animation for #MMIW letters

### PR DESCRIPTION
**Pull request by callum-code**
@mamaro556 once your code is merged or rebased on this, we'll transition to the intro slides in `transitionToIntroSlides()` and then the code that currently lives in that method will be moved to the end of the intro slides.

### CODE REVIEW

**Comment by callum-code** [comment]
Pending Desk Review...

**Review by Mehequanna — APPROVED** [review]
This looks great! Just a question. Looks great on the phone!

File: iOS/MMIWG2S/ViewControllers/IntroAnimationViewController.swift [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    
+    private func addBackground() {
+        let backgroundView = UIImageView(image: UIImage(named: "face-background"))
+        backgroundView.contentMode = .scaleAspectFit
+        
+        view.addSubview(backgroundView)
+        backgroundView.translatesAutoresizingMaskIntoConstraints = false
+        backgroundView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+        backgroundView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+        backgroundView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
```
[the remark this answers is not in the document]
Comment (callum-code) [reply iOS/MMIWG2S/ViewControllers/IntroAnimationViewController.swift:46]: No reason, it's just my personal preference for code style. I'd be happy to change it to the constraints array to keep things consistent accords the project if you want.

[the remark this answers is not in the document]
Comment (Mehequanna) [reply iOS/MMIWG2S/ViewControllers/IntroAnimationViewController.swift:46]: I've just read it's more performant to do them all at once. Either way works for me. If it's a style thing, then I'll defer to you :)

**Comment by callum-code** [comment]
I'll open another pr when the complete desk review comes in from design.

**Pull request merged**